cmd/yorkie/config/config.go

```diff
@@ -137,3 +144,29 @@ func Delete() error {
 
 	return nil
 }
+
+// ReadConfig read configuration file for viper before runnning command
+func ReadConfig(cmd *cobra.Command, args []string) error {
+	configPathValue, err := configPath()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "get config path: %w", err)
+		os.Exit(1)
+	}
+
+	file, err := os.Open(filepath.Clean(configPathValue))
```

Comment:
I had some concerns about this part.

In the existing CLI command, the Load function of config is used, and the Load function checks for the existence of the config path and config.json file and creates one if it doesn't exist.

viper.ReadInConfig() primarily reads the configuration values based on the configname, configtype, and config path set in commands.go.

**During this process, for users who have installed the yorkie program for the first time, the `config.json` does not exist in the `~/.yorkie` directory. Therefore, if simply call `viper.ReadInConfig()`, an error occurs.

To resolve this, I utilized the logic of the Load function from config. Do you think there's a need to modify the logic, or would it be better to remove the duplicated logic from the Load method?**

(Additionally, the statement `path.Join(os.Getenv("HOME"), ".yorkie")` seems not to properly find the HOME directory in a Windows environment. Should this also be taken into consideration?)

Reply:
> Do you think there's a need to modify the logic, or would it be better to remove the duplicated logic from the Load method?

It will be better to refactor duplicate codes in `Load()` and `ReadConfig()`.

Also, since `ReadConfig()`name is confusing with `Load()`, how about changing this function name to something like `Preload()`?

> (Additionally, the statement path.Join(os.Getenv("HOME"), ".yorkie") in commands.go seems not to properly find the HOME directory in a Windows environment. Should this also be taken into consideration?)

We need to cover Windows as well. We might need to code like this: https://gist.github.com/miguelmota/f30a04a6d64bd52d7ab59ea8d95e54da

Reply:
Thank you for the feedback. I will make the changes.